appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { DialogResponses } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+
+/**
+ * Appends the deploySubpath setting if the target path matches the root of a workspace folder
+ * If the targetPath is a sub folder instead of the root, leave the targetPath as-is and assume they want that exact folder used
+ */
+export async function getDeployFsPath(targetPath: string, extensionPrefix: string): Promise<string> {
```

Comment:
If your plan is to only copy this function, you need to at least name it accurately (as it used to be): 
https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/deploy/getDeployFsPath.ts#L39

Reply:
I mean, it still gets the deployFsPath 😬